source/documentation/values/calculations.md

```diff
@@ -225,8 +283,23 @@ combined at build time, so `min(12px % 10, 10%)` will throw an error.
     padding-left: max($padding % 10, 20px);
     padding-right: max($padding % 10, 20px);
   }
+
+  .post-image {
+    // Since these round() calls are valid calculation expressions, they're
+    // parsed as calculations.
+    padding-left: round(nearest, $number, $step);
+    padding-right: round($number + 10px);
+    padding-bottom: round($number + 10px, $step + 10%);
+  }
```

Comment:
These examples should be under the headings for their respective functions, not under `min()` and `max()`.

Reply:
In a [previous comment](https://github.com/sass/sass-site/pull/738#discussion_r1236009723) you mentioned we should have all the examples together, should I change it back to how it was?

Reply:
Ah, sorry for the inconsistency. I think there are two possible ways to go here:

1. Have thorough examples with multiple properties, like you do here, for each function.
2. Have a single style rule (such as `.post-image`) that uses each function _once_.

The way it is now is strange because the example is already visually separated into several distinct blocks, but those blocks don't correspond to the overarching structure of the page.

Reply:
I think the first option would be better, just to provide more examples. :) Lmk your thoughts